Project1/AttributeList.h

```diff
@@ -5,7 +5,7 @@
 #include "Enum.h"
 #include "EnumContainer.h"
 
-enum class _ATTRIB_TYPES { STR, MAG, SKL, SPD, CON, CHA, DEF, RES, ENUM_END };
+enum class _ATTRIB_TYPES { STR, MAG, SKL, SPD, CON, CHA, DEF, RES, MOV, ENUM_END };
```

Comment:
I don't think this should go into AttributeList; no one's going to have MOV growth rates and it's not used for combat. I think it belongs in its own Component<Unit> with the Unit's movement types. I could see an argument for putting both in Stats, but I think that violates Single Responsibility Principle.